### ISSUE TITLE
Wait no longer needed in cypress tests

### ISSUE THREAD

**Issue by Bdegraaf1234** [RADAR-base/ManagementPortal#722]
We have a number of flaky cypress tests, for which we use the cy wait function.

However, it seems that after #716 added retries, this should not be necessary anymore.

We should remove the calls to cy.wait where possible.

**Comment by Bdegraaf1234** [comment]
Unable to find a better way, some efforts are described in #723 but for now we cy.wait()...
